fibonacci.py

```diff
@@ -0,0 +1,9 @@
+f1=1

```

Comment:
please don't make it hard coded. take input from user for how many numbers user wants to print